### PR TITLE
LP-2768 Adding FAQs link

### PR DESCRIPTION
**Pull request by javeriaejaz-arbisoft**

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2246/workflows/workflows_id/jobs/2246/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/branding_extension/constants.py (100%)
openedx/adg/lms/branding_extension/helpers.py (100%)
openedx/adg/lms/branding_extension/tests/helpers_tests.py (100%)
-------------
Total:   5 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/lms/branding_extension/constants.py [hunk 1]
```diff
@@ -1,5 +1,9 @@
 """
 Constants for branding extension
 """
+from django.conf import settings
+
 TARGET_BLANK = '_blank'
 TARGET_SELF = '_self'
+
+FAQs_LINK = f'{settings.SUPPORT_LINK}/hc/en-us/sections/360010666551-FAQ'
```
Comment (MunamMubashir-arbisoft) [inline openedx/adg/lms/branding_extension/constants.py:9]: Are we sure this link will never change?

**Review by MunamMubashir-arbisoft — APPROVED** [review]

File: openedx/adg/lms/branding_extension/constants.py (hunk 1 above)
Comment (MunamMubashir-arbisoft) [inline openedx/adg/lms/branding_extension/constants.py:9]: Please capitalize this completely.

**Review by MunamMubashir-arbisoft — APPROVED** [review]

File: openedx/adg/lms/branding_extension/constants.py (hunk 1 above)
Comment (MunamMubashir-arbisoft) [reply openedx/adg/lms/branding_extension/constants.py:9]: FAQS_LINK

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2247/workflows/workflows_id/jobs/2247/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/branding_extension/constants.py (100%)
openedx/adg/lms/branding_extension/helpers.py (100%)
openedx/adg/lms/branding_extension/tests/helpers_tests.py (100%)
-------------
Total:   5 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/lms/branding_extension/constants.py (hunk 1 above)
Comment (javeriaejaz-arbisoft) [reply openedx/adg/lms/branding_extension/constants.py]: There could be Arabic link later on but we decided to work with only link for now.

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2248/workflows/workflows_id/jobs/2248/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/branding_extension/constants.py (100%)
openedx/adg/lms/branding_extension/helpers.py (100%)
openedx/adg/lms/branding_extension/tests/helpers_tests.py (100%)
-------------
Total:   5 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by annas-usman — APPROVED** [review]

**Pull request merged**